test/indexer-test.js

```diff
@@ -313,9 +313,12 @@ describe('Indexer', function() {
         maxTxs: 10
       });
 
-      await assert.asyncThrows(async () => {
+      await assert.rejects(async () => {
```

Comment:
Nice to have this without requiring node > v10

Reply:
Yep